locales/en/kafka.json

```diff
@@ -75,5 +75,17 @@
   "topic": {
     "topic": "Topic"
   },
-  "will_expire": "This instance will expire in <time>{{time}}</time>"
+  "will_expire": "This instance will expire in <time>{{time}}</time>",
+  "serviceAccount": {
```

Comment:
I'd move this to a service-account.json file. SAs are used across MAS, which isn't limited to Kafka